code/modules/projectiles/guns/projectile/automatic/blackguard.dm

```diff
@@ -14,7 +14,7 @@
 	load_method = SINGLE_CASING|MAGAZINE
 	mag_well = MAG_WELL_HRIFLE|MAG_WELL_DRUM
 	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASTIC = 5, MATERIAL_STEEL = 25, MATERIAL_GLASS = 5)
-	zoom_factor = 0.5 //5 more tiles
+	zoom_factor = 1 // In line with the Longarm omnirifle, less than the Osprey, and half of specialized sniper rifles like the Scout. - Seb
```

Comment:
```suggestion
	zoom_factor = 0.5 //3 more tiles
```

Reply:
blackguard isnt a longarm or a scout

Reply:
The scout has a **x2 scope**. Bat'ko has a **x1.8 scope**. This is x1 scope, a literal ACOG tier. On a marskman rifle, comparable to the Longarm. For another comparison, even with this change, a laser tag sniper has more range than this at x1.2 (9 tiles).

The Blackguard by all definitions is a literal longarm since it's already an omni platform as well as having a long barrel, that's also the reason it has a brace penalty (higher than the Longarm) and a bayonet, and it's classified as "marksman" yet currently has a scope equal in utility as the Soteria PDW, which is egregious for something labeled a "marksman" rifle. It does not fulfill its role at all with such a short ranged scope.

This rifle falls inferior to an STS-40 with an ACOG scope by all means (STS also more easily modded), which can be achieved with half the teleport points and a printed ACOG scope or guild scope for far less hassle, that also takes drums in its magwell. There is currently no justification to get a Blackguard from the teleporter if you can achieve the same levels of reliability with an STS-40 at half its power cost, right now it is totally not viable with it costing a whopping 1000 power to order for just 0.25 more damage multiplier and a bayonet.

Reply:
Im telling you as the person that made the gun, its not meant to be a longarm or a scout

Reply:
Why call it a _"marksman"_ rifle then since it has a 3 tiles scope that's the equivalent of squinting your eyes...

Also, wasn't it originally meant to have 5 tiles of zoom? Getting it down to 3 would be a nerf, if anything.